include/debug/stack_size_analyzer.rst

```diff
@@ -0,0 +1,63 @@
+.. _stack_size_analyzer:
+
+Stack size analyzer
+###################
+
+Stack size analyzer module enables all the Zephyr options required to track
+the stack usage.
+Analyze is performed on demand when the application calls
+:cpp:func:`stack_size_analyzer_run` or :cpp:func:`stack_size_analyzer_print`.
+
+Sample code
+***********
+
+The usage example below would print stacks statistics every 5 seconds.
+It can be added to most examples without modyfing any other code.
```

Comment:
```suggestion
It can be added to most examples without modifying any other code.
```

Reply:
This part would be removed in a moment.,